chromium_src/third_party/tflite_support/BUILD.gn

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//third_party/flatbuffers/flatbuffer.gni")
+
+flatbuffer("encoder_config") {
```

Comment:
I don't know if we really have a policy around what should go in chromium_src in terms of things like BUILD.gn. @brave/chromium-src-reviewers ?

Reply:
moved to `brave/third_party` in 314b991bfc15e85cb209abc6cd87f9ed6c3496ac